source/options.tsx

```diff
@@ -95,6 +95,12 @@ async function updateStorageUsage(area: 'sync' | 'local'): Promise<void> {
 async function validateToken(): Promise<void> {
 	reportStatus({});
 	const tokenField = select('input[name="personalToken"]')!;
+
+	if (tokenField.value.startsWith('github_pat_')) {
+		// Validation not supported yet https://github.com/refined-github/refined-github/issues/6092
+		return;
```

Comment:
Validation fails:


<img width="541" alt="Screenshot 5" src="https://github.com/refined-github/refined-github/assets/1402241/8c6a4c24-706e-442a-a9d7-96967a5c345f">


So this line of code stops it. This is the result:

<img width="541" alt="Screenshot 4" src="https://github.com/refined-github/refined-github/assets/1402241/c93f2704-49f0-4fc0-8e0d-c5ec566bc14e">